### DISCUSSION TITLE
Reduce Window Border Thickness

### DISCUSSION THREAD

**Discussion by dnut** [zen-browser/desktop#3452] [Ideas]
It would be nice if the GUI didn't have such thick borders or rounded corners. Not only is it ugly, but it reduces the usable size of your monitor.

**Comment by Gambloide** [comment]
Seconded. I understand the borders are there to provide a manageable activation zone for having the tool- and side-/tab- bar slide into view on hover.

However, as someone who dislikes that hover behavior and just wants the bars to stay away unless toggled via keyboard, I also perceive them as a downside.

For now, I am using https://zen-browser.app/mods/bfcc400a-4ecb-4752-bfd2-a68f116a2722/ to at least remove them in windows not using split tabs.

There are other mods also removing the borders, but in my testing the results were not consistent, likely due to breaking changes since the mods were made.

**Reply by dnut** [reply]
> I understand the borders are there to provide a manageable activation zone for having the tool- and side-/tab- bar slide into view on hover.

That's a good point. But it doesn't apply to the borders that do not serve as activation zones. For example the bottom of the window.

When a panel is hidden, and it needs an activation zone, then that specific border can thicken to meet the demand. Otherwise all borders could remain narrow.

**Comment by nktnet1** [comment]
1. Go to about:config
1. Look for zen.theme.content-element-separation
1. Set the value to 1, or however small you want the padding to be (default is 8)

Answered on reddit:
- https://www.reddit.com/r/browsers/comments/1erv3mv/comment/moqxdjc

**Reply by KenzKD** [reply]
This still works. I have set mine to 0

**Comment by Emby01** [comment]
As of `1.16.2b` fullscreening while in compact mode removes the borders!

**Reply by KenzKD** [reply]
This does work, but in fullscreen mode you cannot drag the window around. 

You have to exit back to the normal maximized window and only then can you drag the window around. 

Setting the zen.theme.content-element-separation to 0 allows you to remove the window borders without even having to go into fullscreen mode, which makes it more convenient if you like having this style all the time.
